source/guides/references/changelog.md

```diff
@@ -2,7 +2,7 @@
 title: Changelog
 ---
 
-## 3.1.0
+# 3.1.0
```

Comment:
I changed all of the version headings to be `#` (`h1`).

Previously all of the version headings starting with `1.0.0` were `##` (`h2`) which resulted in the MenuSpy code only detecting headings >= `1.0.0`. As you can see in the following screenshot, MenuSpy is only finding the 25 versions  >= `1.0.0`.

<img width="689" alt="screen shot 2018-08-18 at 11 03 23" src="https://user-images.githubusercontent.com/1855109/44308215-07c5bc80-a3a9-11e8-8f72-2f69eeebc826.png">

You can see this in action by [surfing to the current Changelog](https://docs.cypress.io/guides/references/changelog.html#1-0-0)... you can scroll down past `1.0.0` to older versions but the menu won't advance beyond `1.0.0` because that's the last entry in that collection.

After making the change MenuSpy now finds all of the versions.

<img width="383" alt="screen shot 2018-08-18 at 11 08 58" src="https://user-images.githubusercontent.com/1855109/44308224-4491b380-a3a9-11e8-8b77-fb0750b57435.png">

You can now scroll past `1.0.0` and the menu will advance correctly.

My rationale for changing all the version headings to `#` is that each version is a top-level thing. Happy to change 'em  -- or see 'em changed -- to `##`: it's the consistency that's key to getting the MenuSpy behaviour.